data/json/items/armor.json

```diff
@@ -7322,11 +7322,12 @@
         "name" : "suitcase",
         "weight" : 5000,
         "color" : "red",
-        "covers" : ["ARMS", "HANDS", "LEGS"],
+        "covers" : ["ARM_EITHER", "HAND_EITHER", "LEG_EITHER"],
```

Comment:
This halves the overall encumbrance, you should double the encumbrance to compensate for that.